var/spack/repos/builtin/packages/damaris/package.py

```diff
@@ -13,72 +13,83 @@ class Damaris(CMakePackage):
-    version('1.8.1',  tag='v1.8.1')
-    version('1.8.0',  tag='v1.8.0')
-    version('1.7.1',  tag='v1.7.1')
-    version('1.7.0',  tag='v1.7.0')
-
-    variant('fortran',  default=True,  description='Enables Fortran support')
-    variant('hdf5',     default=False, description='Enables the HDF5 storage plugin')
-    variant('static',   default=False, description='Builds a static version of the library')
-    variant('catalyst', default=False, description='Enables the Catalyst visualization plugin')
-    variant('visit',    default=False, description='Enables the VisIt visualization plugin')
-    variant('examples', default=False,
-description='Enables compilation and installation of the examples code')
-    variant('docs',     default=False, description='Enables the building of dOxygen documentation')
-    variant('python',   default=False,
-description='Enables building of Python enabled Damaris library using Boost::python')
-
-    depends_on('mpi')
-    depends_on('cmake@3.18.0:', type=('build'))
-    depends_on('boost+thread+log+filesystem+date_time @1.67:')
-    depends_on('xsd')
-    depends_on('xerces-c')
-    depends_on('hdf5@1.8.20:', when='+hdf5')
-    depends_on('paraview+python+mpi+development_files', when='+catalyst')
-    depends_on('visit+mpi', when='+visit')
-    depends_on('boost+thread+log+filesystem+date_time+python+numpy @1.67:', when='+python')
+    maintainers("jcbowden")
+
+    version("master", branch="master")
+    version("1.9.0",  tag="v1.9.0")
+    version("1.8.2",  tag="v1.8.2")
+    version("1.8.1",  tag="v1.8.1")
+    version("1.8.0",  tag="v1.8.0")
+    version("1.7.1",  tag="v1.7.1")
+    version("1.7.0",  tag="v1.7.0")
+    version("1.6.0", tag="v1.6.0", deprecated=True)
+    version("1.5.0", tag="v1.5.0", deprecated=True)
+    version("1.3.3", tag="v1.3.3", deprecated=True)
+    version("1.3.2", tag="v1.3.2", deprecated=True)
+    version("1.3.1", tag="v1.3.1", deprecated=True)
+
+    variant("fortran", default=True, description="Enables Fortran support")
+    variant("hdf5", default=False, description="Enables the HDF5 storage plugin")
+    variant("static", default=False, description="Builds a static version of the library")
+    variant("catalyst", default=False, description="Enables the Catalyst visualization plugin")
+    variant("visit", default=False, description="Enables the VisIt visualization plugin")
+    variant(
+        "examples",
+        default=False,
+        description="Enables compilation and installation of the examples code",
+    )
+    variant("docs", default=False, description="Enables the building of dOxygen documentation")
+    variant(
+        "python",
+        default=False,
+        description="Enables building of Python enabled Damaris library using Boost::python",
+    )
+
+    depends_on("xsd")
+    depends_on("xerces-c")
+    depends_on("mpi")
+    depends_on("cmake@3.18.0:", type=("build"))
+    depends_on("boost" "+thread+log+filesystem+date_time" "@1.67:")
```

Comment:
The extra double quotes are awkward here and below.

```suggestion
    depends_on("boost+thread+log+filesystem+date_time @1.67:")
```